examples/thoughts/ext/TagsSidebar.js

```diff
@@ -14,19 +14,20 @@ const TagsSidebar = (props) => {
   return (
     <div className="tags-sidebar">
       <Link to="/" className="new-thought-link"> New thought </Link>
-      <Link to={`/thoughts`}
-            className={`tags-sidebar-tag tags-sidebar-tag-all ${props.active === '_all' ? 'active' : ''}`}
-            style={{ color: 'black' }}
-            key="_all">
-        all
-      </Link>
+      <Tag
+        name={'all'}
+        href={'/thoughts'}
+        isActive={props.active === '_all'}
+        overrideKey={'_all'}
```

Comment:
You still have both `overrideKey` and `key` here -> I believe you said you will remove `overrideKey`?

Reply:
Yep, my bad, I accidentally didn't save this file before committing my changes. Fixed now.